CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ All notable changes to `src-cli` are documented in this file.
 
 ### Fixed
 
+- `src extensions copy` now removes the `url` property from extension manifests that points to the sourcegraph.com extension bundle, enabling use for air-gapped instances.
```

Comment:
Not sure if this too persnickety, or if you guys have strong opinions on how we communicate changes in the changelog, but...

Is `sourcegraph.com` always going to be the prefix of the `BundleURL` for `copy`'d extensions? Could one theoretically copy an extension from some other sourcegraph instance (e.g. cse-aws-test.sgdev.org copies from k8s.sgdev.org)? Very edge-casey, I suppose.

I guess I'd prefer language that's more like "resets the `BundleURL`" or something, if it's a possibility that extensions can be copied from not-sourcegraph.com instances.

Deferring entirely to your guys' judgement, just wanted to express my concerns.

Reply:
Good thinking! For now, `copy` is [hardcoded](https://sourcegraph.com/github.com/sourcegraph/src-cli@tjk/ext-copy-remove-url/-/blob/cmd/src/extensions_copy.go?L23%3A65=) to use `sourcegraph.com`

Reply:
Hey @tjkandala , the link you provided doesn't work for me anymore (I assume because the branch got deleted), but I'm assuming you intended to refer to [this LOC](https://github.com/sourcegraph/src-cli/blob/1c260bce6d91725ef2c17399cd5abdf1972be238/cmd/src/extensions_copy.go#L75)?